assignments/objects.js

```diff
@@ -19,27 +19,80 @@ const example = {
 
 // Write your intern objects here:
 
+const intern1 = {
+  "id":1,
+  "email":"mmelloy0@psu.edu:",
+  "fName":"Mitzi",
+  "gender":"F",
+}
+
+const intern2 = {
+  "id":2,
+  "email":"kdiben1@tinypic.com",
+  "fName": "Kennan",
+  "gender":"M",
+}
+const intern3 = {
+  "id":3,
+  "email":"kmummery2@wikimedia.org",
+  "fName":"Keven",
+  "gender": "M",
+}
+const intern4 = {
+  "id":4,
+  "email": "gmartinson3@illinois.edu",
+  "fName":"Gannie",
+  "gender":"M",
+}
+const intern5 = {
+  "id":5,
+  "email": "adaine5@samsung.com",
+  "fName":"Antonietta",
+  "gender":"F",
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(intern1.fName);
+
 // Kennan's ID
 
+console.log(intern2.id);
+
 // Keven's email
 
+console.log(intern3.email);
+
 // Gannie's name
 
+console.log(intern4.fName);
+
+
 // Antonietta's Gender
 
+console.log(intern5.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+intern2.speak = function(){ 
+  return "Hello, my name is Kennan!";
```

Comment:
Fantastic job! 

Also don't be afraid to use your extension here to help with spacing.